operations/list.go

```diff
@@ -257,30 +271,52 @@ func listParameters(ctx context.Context, confPath, project, filename string) err
 	return nil
 }
 
-func listAliases(ctx context.Context, confPath, project, filename string) error {
+func listTriggerAliases(ctx context.Context, confPath, project string) error {
 	conf, err := NewClientSettings(confPath)
 	if err != nil {
 		return errors.Wrap(err, "problem loading configuration")
 	}
 	comm := conf.setupRestCommunicator(ctx)
 	defer comm.Close()
 
-	var aliases []model.ProjectAlias
+	var aliases []string
 
 	if project != "" {
-		aliases, err = comm.ListAliases(ctx, project)
+		aliases, err = comm.ListPatchTriggerAliases(ctx, project)
 		if err != nil {
 			return err
 		}
-	} else if filename != "" {
-		project, err := loadLocalConfig(filename)
+	} else {
+		return errors.New("no project specified")
+	}
+
+	for _, alias := range aliases {
+		fmt.Printf("%s\n", alias)
+	}
+
+	return nil
+}
+func deprecatedListAliases(ctx context.Context, confPath, project, patchAliasesFlagName string) error {
+	err := listPatchAliases(ctx, confPath, project)
+
+	fmt.Printf("\n --aliases is deprecated and will be removed soon. Please use --%s instead. \n", patchAliasesFlagName)
+	return err
+}
+func listPatchAliases(ctx context.Context, confPath, project string) error {
+	conf, err := NewClientSettings(confPath)
+	if err != nil {
+		return errors.Wrap(err, "problem loading configuration")
+	}
+	comm := conf.setupRestCommunicator(ctx)
+	defer comm.Close()
+
+	var aliases []model.ProjectAlias
+
+	if project != "" {
```

Comment:
would it be clearer to return here if `project == ""`? (the same comment applies in `listTriggerAliases`)

Reply:
most definitely! thanks!